tests/conftest.py

```diff
@@ -112,14 +112,15 @@ def bootstrap_config(host):
 
 
 @pytest.fixture
-def registry_address(host):
+def registry_address(host, version):
     with host.sudo():
         registry_json = host.check_output(
-            "salt-call pillar.get metalk8s:endpoints:repositories --out json"
+            "salt-call --out json slsutil.renderer string='"
+            "{% from \"metalk8s/map.jinja\" import repo with context %}"
+            "{{ repo.registry_endpoint }}' "
```

Comment:
Can't we just read from `defaults.yaml`?

Reply:
Yes btw I don't think we will override it in map.jinja one day but for me it's better to really get the final value :)